Makefile

```diff
@@ -50,11 +50,11 @@ DOCKER_CMD_INT := $(DOCKER_CMD_BASE) -it $(DEV_IMAGE)
 # See https://github.com/Azure/open-service-broker-azure/issues/100
 BASE_IMAGE_NAME        = azure-service-broker
 
-RC_IMAGE_NAME          = $(DOCKER_REPO)$(BASE_IMAGE_NAME):$(GIT_VERSION)
-RC_MUTABLE_IMAGE_NAME  = $(DOCKER_REPO)$(BASE_IMAGE_NAME):canary
+RC_IMAGE_NAME          = $(DOCKER_REGISTRY)$(DOCKER_REPO)$(BASE_IMAGE_NAME):$(GIT_VERSION)
+RC_MUTABLE_IMAGE_NAME  = $(DOCKER_REGISTRY)$(DOCKER_REPO)$(BASE_IMAGE_NAME):canary
```

Comment:
Nit: I know `DOCKER_REPO` is a name I chose a long, long time ago, but in retrospect, it was wrong. Now could be a good time to correct it. Technically, in a Docker registry, a repo is a store for a single family of images (i.e. same name, different tags).

User or org names containing multiple repos are probably more accurately called "registry namespaces."

In more recent projects I've worked on, I've changed this env var to `DOCKER_REGISTRY_NAMESPACE` for clarity.

I'm not suggesting you _have_ to do this. Just raising awareness of the inaccuracy of the term `DOCKER_REPO` here and how it can be corrected if you wish.

Reply:
I've also included this snippet in some recent projects because it eliminates the unintuitive need for including a trailing slash in registry and namespace env vars:

```
ifdef DOCKER_REGISTRY
	DOCKER_REGISTRY := $(DOCKER_REGISTRY)/
endif

ifdef DOCKER_REGISTRY_NAMESPACE
	DOCKER_REGISTRY_NAMESPACE := $(DOCKER_REGISTRY_NAMESPACE)/
endif
```

Reply:
Thanks for the suggestions. Totally agree!